bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCBridgeHandler.java

```diff
@@ -256,20 +295,26 @@ public void dispose() {
         logger.debug("Dispose LOGO! bridge handler.");
         super.dispose();
 
+        final var rtcJob = this.rtcJob;
         if (rtcJob != null) {
             rtcJob.cancel(false);
-            rtcJob = null;
-            logger.info("Destroy RTC job for {}.", config.get().getAddress());
+            logger.info("Destroy RTC job for {}.", config.getAddress());
         }
+        this.rtcJob = null;
 
+        final var readerJob = this.readerJob;
         if (readerJob != null) {
             readerJob.cancel(false);
-            readerJob = null;
-            logger.info("Destroy reader job for {}.", config.get().getAddress());
+            logger.info("Destroy reader job for {}.", config.getAddress());
```

Comment:
or even trace
```suggestion
            logger.debug("Destroy reader job for {}.", config.getAddress());
```

Reply:
Allow edits and access to secrets by ma

Reply:
see above comment. info should not be used for debugging issues.